components/crud-web-apps/jupyter/frontend/src/app/pages/form/form-default/form-default.component.ts

```diff
@@ -106,7 +106,49 @@ export class FormDefaultComponent implements OnInit, OnDestroy {
 
     // Use the custom image instead
     if (notebook.customImageCheck) {
+      // Remove unnecessary images from payloaf
+      delete notebook.imageVSCode;
```

Comment:
IIUC we always want to remove these extra fields right? In this case we can do this outside of all the if statements once.

Also since we will expect the frontend to only send the `serverType`, and not the headers/uri-rewrite, this section will be significantly simplified.

Reply:
There isn't really a reason to remove them, but I thought it would be better not to send unused stuff in the request. Indeed this can most likely be simplified.

Reply:
Oh maybe I didn't express it clearly, I agree that the frontend should not be sending the `imageVSCode` and `imageRStudio` fields. 

My comment is to write the `delete notebook.imageVSCode` [ and for rstudio ] commands in one place at the top and not do it in multiple places inside the different ifs

Reply:
Sorry, I think my comment wasn't clear. We are completely on the same page ;).